common/src/main/java/com/gregtechceu/gtceu/data/lang/MachineLang.java

```diff
@@ -412,14 +442,14 @@ protected static void init(RegistrateLangProvider provider) {
         multiLang(provider, "gtceu.multiblock.monitor_screen.tooltip", "The GUI can be opened with a right-click of a screwdriver.", "The proxy mode of Digital Interface Cover can delegate machines' capabilities and GUI. (Yes, you can connect pipes directly on the screen.)", "The screen also supports plugins.");
     }
 
-    private static void standardTooltips(RegistrateLangProvider provider, String root, String lowTier, String midTier, String highTier) {
-        provider.add("%s.%s.tooltip".formatted(root, "lv"), "§7%s".formatted(lowTier));
-        provider.add("%s.%s.tooltip".formatted(root, "mv"), "§7%s".formatted(lowTier));
-        provider.add("%s.%s.tooltip".formatted(root, "hv"), "§7%s".formatted(lowTier));
-        provider.add("%s.%s.tooltip".formatted(root, "ev"), "§7%s".formatted(lowTier));
-        provider.add("%s.%s.tooltip".formatted(root, "iv"), "§7%s".formatted(midTier));
-        provider.add("%s.%s.tooltip".formatted(root, "luv"), "§7%s".formatted(midTier));
-        provider.add("%s.%s.tooltip".formatted(root, "zpm"), "§7%s".formatted(midTier));
-        provider.add("%s.%s.tooltip".formatted(root, "uv"), "§7%s".formatted(highTier));
+    private static void standardTooltips(RegistrateLangProvider provider, String root, String machine, String lowTier, String midTier, String highTier) {
```

Comment:
You may want to remove the `root` parameter and directly inline it into the format string, as it's always `"gtceu.machine."` right now.